2024-11-03 22:49:56 [ERROR] Error: Cannot find module './MongoSpecificAnalyser.'
Require stack:
- D:\mongomath\testing.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (D:\mongomath\testing.js:2:20)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
2024-11-03 23:49:08 [ERROR] ReferenceError: ex is not defined
    at Object.<anonymous> (D:\mongomath\testing.js:71:2)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 00:11:44 [ERROR] TypeError: MongooseAnalyzer is not a constructor
    at new MongoMath (D:\mongomath\testing.js:10:27)
    at Object.<anonymous> (D:\mongomath\testing.js:79:19)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 00:12:32 [ERROR] TypeError: MongooseAnalyzer is not a constructor
    at new MongoMath (D:\mongomath\testing.js:10:27)
    at Object.<anonymous> (D:\mongomath\testing.js:79:19)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 00:12:39 [ERROR] Error: Cannot find module './globaldb'
Require stack:
- D:\mongomath\testing.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (D:\mongomath\testing.js:3:26)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
2024-11-04 08:10:58 [ERROR] TypeError: mongoMath.dataAnalyzer is not a function
    at D:\mongomath\testing.js:5:21
    at Object.<anonymous> (D:\mongomath\testing.js:14:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 08:11:17 [ERROR] TypeError: mongoMath.dataAnalyzer is not a function
    at D:\mongomath\testing.js:5:21
    at Object.<anonymous> (D:\mongomath\testing.js:14:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:15:17 [ERROR] Error: Analysis error: Not connected to database. Call connect() first.
    at mongomath.dataAnalyzer (D:\mongomath\mongomath.js:76:13)
    at D:\mongomath\testing.js:5:19
    at Object.<anonymous> (D:\mongomath\testing.js:14:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:21:54 [ERROR] ReferenceError: result is not defined
    at D:\mongomath\testing.js:10:26
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-11-04 10:27:43 [ERROR] ReferenceError: analysis2 is not defined
    at D:\mongomath\testing.js:32:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2024-11-04 10:36:23 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.getPerformanceMetrics (D:\mongomath\mongomath.js:165:37)
    at D:\mongomath\testing.js:20:37
    at Object.<anonymous> (D:\mongomath\testing.js:33:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:39:56 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.getPerformanceMetrics (D:\mongomath\mongomath.js:165:37)
    at D:\mongomath\testing.js:20:37
    at Object.<anonymous> (D:\mongomath\testing.js:33:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:41:12 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.getPerformanceMetrics (D:\mongomath\mongomath.js:165:37)
    at D:\mongomath\testing.js:20:37
    at Object.<anonymous> (D:\mongomath\testing.js:33:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:42:41 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.analyzeDatabaseComplete (D:\mongomath\mongomath.js:192:37)
    at D:\mongomath\testing.js:7:35
    at Object.<anonymous> (D:\mongomath\testing.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:43:12 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.analyzeDatabaseComplete (D:\mongomath\mongomath.js:192:37)
    at D:\mongomath\testing.js:7:35
    at Object.<anonymous> (D:\mongomath\testing.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:43:30 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.analyzeDatabaseComplete (D:\mongomath\mongomath.js:192:37)
    at D:\mongomath\testing.js:7:35
    at Object.<anonymous> (D:\mongomath\testing.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:44:42 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.analyzeDatabaseComplete (D:\mongomath\mongomath.js:192:37)
    at D:\mongomath\testing.js:7:35
    at Object.<anonymous> (D:\mongomath\testing.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
2024-11-04 10:46:34 [ERROR] Error: Database connection not established. Please connect manually or use selfloading=true
    at MongoMath.ensureConnection (D:\mongomath\mongomath.js:20:13)
    at MongoMath.analyzeDatabaseComplete (D:\mongomath\mongomath.js:192:37)
    at D:\mongomath\testing.js:7:35
    at Object.<anonymous> (D:\mongomath\testing.js:13:3)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49
